MapboxCoreNavigation/RouteController.swift

```diff
@@ -363,25 +372,16 @@ extension RouteController: CLLocationManagerDelegate {
             guard let route = route else { return }
             
             if route.legs[0].steps[1].expectedTravelTime <= 70 {
-                let maneuverLocation = CLLocation(coordinate: currentUpcomingManeuver.maneuverLocation, altitude: 0, horizontalAccuracy: 1, verticalAccuracy: 1, course: currentUpcomingManeuver.finalHeading ?? currentUpcomingManeuver.initialHeading ?? location.course, speed: 10, timestamp: Date())
+                let course = currentUpcomingManeuver.finalHeading ?? currentUpcomingManeuver.initialHeading ?? location.course
+                let maneuverLocation = CLLocation(coordinate: currentUpcomingManeuver.maneuverLocation, altitude: 1, horizontalAccuracy: 1, verticalAccuracy: 1, course: course, speed: 1, timestamp: Date())
                 
-                strongSelf.getDirections(from: maneuverLocation, completion: { (route, error) in
-                    guard let route = route else { return }
+                strongSelf.getDirections(from: maneuverLocation) { (route, error) in
+                    guard let newRoute = route else { return }
```

Comment:
`newRoute` is just an unwrapped version of `route`, so it’s fine to shadow the original variable and call the new local variable `route` as well.